### PR TITLE
docs: image fixes

### PR DESCRIPTION
**Pull request by clement-bonnet**

### CODE REVIEW

File: docs/environments/job_shop.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # JobShop Environment
 
 <p align="center">
-        <img src="../env_anim/job_shop_annotated.gif" width="600"/>
+        <img src="../img/job_shop_annotated.gif" width="600"/>
```
Comment (dluo96) [inline docs/environments/job_shop.md:4]: Why the switch?

**Review by dluo96 — APPROVED** [review]
LGTM!

File: docs/environments/job_shop.md (hunk 1 above)
Comment (clement-bonnet) [reply docs/environments/job_shop.md:4]: We want to keep in `env_anim` one GIF per environment. JobShop already has such a gif called `job_shop.gif`. Hence we either remove one or move this one to the more general `img` folder that does not have restrictions.

Comment (dluo96) [reply docs/environments/job_shop.md:4]: Agreed!

**Pull request merged**